### PR TITLE
Feature/automatic releases

### PR DESCRIPTION
**Pull request by kotae4**
adds automated, zipped releases that users can download even without a github account

* dunno if repository secrets will be wiped during merge, probably not?
* can invoke discord webhook and send it ${{ steps.automatic_releases.outputs.upload_url }}, but i think that requires changes on your side.
* can remove the uploading of artifacts entirely once discord webhooks are adjusted, unless there's a reason to keep them?

### CODE REVIEW

**Comment by kotae4** [comment]
see https://github.com/kotae4/AmongUsMenu/releases for output

**Review by ChaseBrand — COMMENTED** [review]
It looks good to me, but we'll wait on Osome or nullptr's approval before changing the actions. We might have to change some things internally on either the bot or webhook for it.

**Review by v0idp — APPROVED** [review]

**Pull request merged**
